proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -360,3 +362,16 @@ message ResendReplicationTasksRequest {
 
 message ResendReplicationTasksResponse {
 }
+
+message GetTaskQueueTasksRequest {
```

Comment:
GetTaskQueueTasksRequest -> ListTaskQueueTasksRequest & add a pagination token?

Reply:
afaict this also requires changes in persistence right so it returns a page token? i'm opening a ticket for refactoring